engine/src/main/java/io/camunda/zeebe/engine/transport/InterPartitionCommandSender.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.engine.transport;
+
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.intent.Intent;
+import io.camunda.zeebe.util.buffer.BufferWriter;
+
+/**
+ * Supports sending arbitrary commands to another partition. Sending may be unreliable and fail
+ * silently, it is up to the caller to detect this and retry.
+ */
+public interface InterPartitionCommandSender {
+  String TOPIC_PREFIX = "inter-partition-";
```

Comment:
:wrench: Defining the topic here doesn't sound ideal. It would be good it this is moved to the broker along with the sender/receiver implementation.